CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The format is almost based on [Keep a Changelog](https://keepachangelog.com/en/1
 ### Fixed
 - Allow negative limits (#1275)
 - Use boolean to check bool fields (#1278)
+- Check category label for null (#1282)
```

Comment:
It's not in the released 15.4.0-beta3. please move it to the Unreleased section instead.

Reply:
Corrected.